sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -85,9 +85,9 @@ export class EventDataBatch {
 
 // @public
 export class EventHubClient {
-    constructor(host: string, eventHubName: string, credential: TokenCredential, options?: EventHubClientOptions);
```

Comment:
can you revert the change to this file please? Else Event Hubs CI gets triggered unnecessarily
it only re-orders 1 line and I guess its due the version of node or some underlying tool we are using, because when I run it, it will revert this line.

Reply:
Same goes for service bus and storage or any non key vault related files